MapboxNavigation/Style.swift

```diff
@@ -209,6 +209,27 @@ public class ResumeButton: UIControl {
     }
 }
 
+/// :nodoc:
+@objc(MBDraggableView)
+open class StepListIndicatorView: UIView {
+    
+    // Workaround the fact that UIView properties are not marked with UI_APPEARANCE_SELECTOR
+    @objc dynamic open var gradientColors: [UIColor] = [.gray, .lightGray, .gray]
```

Comment:
Add a didSet that calls `setNeedsLayout` so that it works w/o UIAppearance as well.